js/app/src/api_docs/CodeSnippets.svelte

```diff
@@ -18,19 +27,12 @@
 	let python_code: HTMLElement;
 	let js_code: HTMLElement;
 
+	let has_file_path = endpoint_parameters.some(
+		(param: EndpointParameter) => param.python_type.type === "filepath"
```

Comment:
What if instead of making this change, we wrap the data in `example_inputs` method with the File dataclass? That way it shows up for Video/File as well. Right now this check does not take those components into account (they have nested file types).

Reply:
Ah yeah good catch, will that show up correctly in the snippet if I just wrap in a File()? I'll double check

Reply:
I think so but will be good to double check.